get_latest_configs.sh

```diff
@@ -0,0 +1,14 @@
+#bin/bash
+
+wget -q -O config.template.yaml https://raw.githubusercontent.com/calzoneman/sync/3.0/config.template.yaml
```

Comment:
If we don't end up going the submodule route, we should add a check here in case the user doesn't have `wget` installed.  Or better yet, try `wget` and `curl` and if neither is installed, prompt the user to install files manually.  Something like this:

```sh
#!/bin/sh

if hash wget 2>/dev/null
then
  DOWNLOAD_CMD="wget -q -O"
elif hash curl 2>/dev/null
  DOWNLOAD_CMD="curl -o"
else
  echo "Could not find wget or curl, please download configuration file manually from https://github.com/calzoneman/sync" >&2
  exit 1
fi
```

Reply:
I changed the way this file works since I added sync as a submodule. Instead it will be copying the files from the sync folder, rather than downloading them.